### PR TITLE
Add Ragna

### PR DESCRIPTION
**Pull request by kklein**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kklein** [comment]
I confirm that I'm willing to be listed as a maintainer.

File: recipes/Ragna/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Ragna-{{ version }}.tar.gz
+  sha256: c4da922760b2ab55cb5b6c9bbe010768ad0f7e1d1f7e0f07fbf9ce6ce42ee84b
+
+build:
+  skip: true  # [py<39]
```
Comment (xhochy) [inline recipes/Ragna/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/Ragna/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.9
+    - aiofiles
+    - emoji
+    - fastapi
+    - httpx
+    - importlib-metadata >=4.6  # [py<310]
```
Comment (xhochy) [inline recipes/Ragna/meta.yaml:31]: It's OK to have this empty dependency for newer Pythons
```suggestion
    - importlib-metadata >=4.6
```

**Comment by kklein** [comment]
friendly ping @conda-forge/core :)

**Review by xhochy — APPROVED** [review]
Please don't ping core but help-python in future

**Pull request merged**